public/faq.php

```diff
@@ -0,0 +1,156 @@
+<?php require_once('includes/header.php'); ?>
+
+<section class="faq-holding">
+    <h1 class="faq-holding-heading">Frequently Asked Questions</h1>
+    <h5 class="faq-holding-text">Are you looking for an answer to your questions about BALL? <br> Here we have compiled an overview of frequently asked questions we receive from our BALLers</h5>
+
+    <section class="faq-holding-form col-lg-6 col-md-11 col-11">
+        <form class="form-inline faq-form">
+            <label class="sr-only" for="search">Search</label>
+            <div class="input-group col-lg-8 col-md-8 col-12">
+              <div class="input-group-prepend">
+                <div class="input-group-text"><img src="./assets/img/icons/search-icon.png" class="input-icon" alt="location icon"></div>
+              </div>
+              <input type="text" class="form-control hold-input" id="search" placeholder="Search for a question...">
+            </div>
+          
+            <div class="input-group col-lg-4 col-md-4 col-12">
+                <button type="submit" class="btn btn-primary btn-large faq-holding-search-btn">Search</button>
+            </div>
+        </form>
+    </section>
+</section>
+
+<section class="faq-category">
+    <h6 class="faq-category-text">or chooose a category to quickly find the help you need</h6>
+
+    <div class="faq-category-slider">
+        <div class="faq-category-slide">
+            <img src="./assets/img/icons/profile-icon.png" class="faq-category-slide-img" alt="Profile">
+            <h5 class="faq-category-slide-text">Profile</h5>
+        </div>
+        <div class="faq-category-slide">
+            <img src="./assets/img/icons/geting-started-icon.png" class="faq-category-slide-img" alt="Profile">
+            <h5 class="faq-category-slide-text">Getting Started</h5>
+        </div>
+        <div class="faq-category-slide">
+            <img src="./assets/img/icons/payment-icon.png" class="faq-category-slide-img" alt="Profile">
```

Comment:
I believe the alt for this should be `Payment Icon`. Kindly fix the other images too.